Source/Libraries/CorruptCore/CorruptCore.cs

```diff
@@ -124,10 +125,10 @@ public static long ErrorDelay
             set => AllSpec.CorruptCoreSpec.Update(RTCSPEC.CORE_ERRORDELAY, value);
         }
 
-        public static BlastRadius Radius
+        public static BlastRadiusAlgorithm Radius
         {
-            get => (BlastRadius)AllSpec.CorruptCoreSpec[RTCSPEC.CORE_RADIUS];
-            set => AllSpec.CorruptCoreSpec.Update(RTCSPEC.CORE_RADIUS, value);
+            get => BlastRadius.GetFromString((string)AllSpec.CorruptCoreSpec[RTCSPEC.CORE_RADIUS]);
+            set => AllSpec.CorruptCoreSpec.Update(RTCSPEC.CORE_RADIUS, value.ToString());
```

Comment:
It would be preferable to store a `BlastRadiusAlgorithm` here instead of a string, however there's no conversion from `System.Func<...>` to `object`

Reply:
You sure? Seems to work fine here https://dotnetfiddle.net/TxfNNa
I know Ceras (the serialization library we use) can serialize delegates but it's disabled due to the obvious security concerns.

Either way, there's only a few ways to approach this. We could technically use reflection but that's not much better of a solution

Reply:
It works fine here:
```
set => AllSpec.CorruptCoreSpec.Update(RTCSPEC.CORE_RADIUS, value);
```

But it doesn't work here:
```
partial[RTCSPEC.CORE_RADIUS] = BlastRadius.Spread; // Cannot convert method group 'Spread' to non-delegate type 'object'. Did you intend to invoke the method?

Reply:
Ah I see. Yeah that's not gonna work.

Reply:
Yeah, I think we'll need to manually serialize/deserialize with some sort of primitive here. Using an `Enum` would be cleaner than using a `string`, so I can switch to that if we end up deciding on the `Func` implementation.